package.json

```diff
@@ -83,14 +77,7 @@
     "@typescript-eslint/eslint-plugin": "4.14.1",
     "@typescript-eslint/parser": "4.14.1",
     "@wojtekmaj/enzyme-adapter-react-17": "0.3.2",
-    "babel-core": "7.0.0-bridge.0",
-    "babel-loader": "^8.2.2",
-    "babel-plugin-add-react-displayname": "0.0.5",
-    "babel-plugin-macros": "2.8.0",
-    "babel-plugin-preval": "5.0.0",
-    "babel-plugin-styled-components": "1.10.7",
-    "babel-plugin-transform-replace-expressions": "0.2.0",
-    "babel-polyfill": "6.26.0",
+    "acorn-jsx": "5.3.1",
```

Comment:
TODO: (Optional) Stub `acorn-jsx` types, similar to what we did for [`@styled-system/prop-types`](https://github.com/primer/components/blob/main/%40types/%40styled-system/prop-types/index.d.ts)